src/platform/tigerlake/include/platform/drivers/alh.h

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
/* ... */ in headers.
Could you also fix this in the alh headers for cnl and icl? Can be done in additional commit.

Reply:
Very well